ot-node.js

```diff
@@ -699,16 +699,16 @@ class OTNode {
                 return;
             }
 
-            const externalId = req.params.replication_id;
-            if (externalId == null) {
+            const replicationId = req.params.replication_id;
+            if (replicationId == null) {
                 log.error('Invalid request. You need to provide replication ID');
                 res.status = 400;
                 res.send({
                     message: 'Replication ID is not provided',
                 });
             } else {
                 const queryObject = {
-                    external_id: externalId,
+                    replicationId,
```

Comment:
@simonovic86 Is this "replicationId: replicationId", identical to just writing "replicationId" ?

Reply:
👍